src/backend/src/routes/tasks.routes.ts

```diff
@@ -8,7 +8,7 @@ const tasksRouter = express.Router();
 tasksRouter.post(
   '/:wbsNum',
   nonEmptyString(body('title')),
-  body('deadline').isDate(),
+  body('deadline').optional().isDate(),
```

Comment:
we should be using the isDate function, not sure if it supports optional values though... Might have to add that